shared/js/ui/templates/site.es6.js

```diff
@@ -41,12 +41,12 @@ module.exports = function () {
             </a>
         </li>
         <li class="site-info__li--toggle padded ${this.model.isWhitelisted ? '' : 'is-active'}">
-            <h2 class="is-transparent site-info__whitelist-status js-site-whitelist-status">
+            <p class="is-transparent site-info__whitelist-status js-site-whitelist-status">
```

Comment:
This was missing "bold" class.

before:
![chrome-before2](https://user-images.githubusercontent.com/985504/117133275-56f3eb80-ada4-11eb-8473-224d5d9faef2.jpg)

after:
![chrome-after2](https://user-images.githubusercontent.com/985504/117133268-55c2be80-ada4-11eb-850e-20558410097b.jpg)